spec/migrate-config-options-spec.js

```diff
@@ -0,0 +1,37 @@
+'use babel'
+
+import migrateConfigOptions from '../src/migrate-config-options'
+import makeSpy from './make-spy'
+
+describe('migreateConfigOptions()', () => {
```

Comment:
typo her

Reply:
damn…

Reply:
I do think it's funny that you also had a typo, in pointing out my typo:

> typo her